src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -132,6 +139,127 @@ public function getCrossingChar()
         return $this->crossingChar;
     }
 
+    /**
+     * Sets crossing characters.
+     *
+     * Example:
+     * <code>
+     * 1---------------2-----------------------2------------------3
+     * | ISBN          | Title                 | Author           |
+     * 8---------------0-----------------------0------------------4
+     * | 99921-58-10-7 | Divine Comedy         | Dante Alighieri  |
+     * | 9971-5-0210-0 | A Tale of Two Cities  | Charles Dickens  |
+     * | 960-425-059-0 | The Lord of the Rings | J. R. R. Tolkien |
+     * 7---------------6-----------------------6------------------5
+     * </code>
+     *
+     * @param string      $cross       Crossing char (see #0 of example)
+     * @param string|null $topLeft     Top left char (see #1 of example), equal to $cross if null
+     * @param string|null $topMid      Top mid char (see #2 of example), equal to $cross if null
+     * @param string|null $topRight    Top right char (see #3 of example), equal to $cross if null
+     * @param string|null $midRight    Mid right char (see #4 of example), equal to $cross if null
+     * @param string|null $bottomRight Bottom right char (see #5 of example), equal to $cross if null
+     * @param string|null $bottomMid   Bottom mid char (see #6 of example), equal to $cross if null
+     * @param string|null $bottomLeft  Bottom left char (see #7 of example), equal to $cross if null
+     * @param string|null $midLeft     Mid left char (see #8 of example), equal to $cross if null
+     *
+     * @return $this
+     */
+    public function setCrossingChars(string $cross, string $topLeft = null, string $topMid = null, string $topRight = null, string $midRight = null, string $bottomRight = null, string $bottomMid = null, string $bottomLeft = null, string $midLeft = null): self
+    {
+        $this->crossingChar = $cross;
+        $this->crossingTopLeftChar = $topLeft ?? $cross;
+        $this->crossingTopMidChar = $topMid ?? $cross;
+        $this->crossingTopRightChar = $topRight ?? $cross;
+        $this->crossingMidRightChar = $midRight ?? $cross;
+        $this->crossingBottomRightChar = $bottomRight ?? $cross;
+        $this->crossingBottomMidChar = $bottomMid ?? $cross;
+        $this->crossingBottomLeftChar = $bottomLeft ?? $cross;
+        $this->crossingMidLeftChar = $midLeft ?? $cross;
+
+        return $this;
+    }
+
+    /**
+     * Gets crossing top right char.
+     *
+     * @return string $crossingTopRightChar
+     */
```

Comment:
You probably added it for consistency, but please remove this docblock as it does not give more info than the signature itself, same for next ones below.

Reply:
Yes, it was for consistency with others methods. Dockblocks removed.